examples/FisherPrice_cu/cufisher_price.cu

```diff
@@ -33,32 +33,35 @@ struct Scoring {
 };
 
 // kernel function that does energy loss or pair production
-__global__ void process(adept::BlockData<MyTrack> *block, Scoring *scor, curandState_t *states)
+__global__ void process(adept::BlockData<MyTrack> *block, Scoring *scor, curandState_t *states, int max_index)
 {
   int particle_index = blockIdx.x * blockDim.x + threadIdx.x;
 
   // check if you are not outside the used block
-  if (particle_index > block->GetNused() + block->GetNholes()) return;
```

Comment:
Indeed this was not correct, in the old approach should read: particle index >= ...